docs/pipelineconfiguration.rst

```diff
@@ -133,3 +133,44 @@ You might toggle the parameter only for manually restarting the service after yo
 start_with_unprocessed = False
 
 ```
+
+SETUP - ESA publishing
+----------------------
+
+processed/generated files by the pipeline should be published to ESA on a regulare base. This is done by the :doc:`publish CLI </html/code_ref/util.html#stix-pipline-cli>` setup a job in the crontab
+
+`crontab -e`
+
+`0 5 * * * cd /home/stixcore/STIXCore/ && /home/stixcore/STIXCore/venv/bin/python /home/stixcore/STIXCore/stixcore/processing/publish.py`
+
+without provided arguments the default values from `stixcore.ini` are used
+
+CLI Interface
+*************
+
+usage: publish.py [-h] [-t TARGET_DIR] [-T TARGET_HOST] [-s SAME_ESA_NAME_DIR] [-d DB_FILE] [-w WAITING_PERIOD]
+                  [-v INCLUDE_VERSIONS] [-l INCLUDE_LEVELS] [-p INCLUDE_PRODUCTS] [-f FITS_DIR]
+
+STIX publish to ESA processing step
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -t TARGET_DIR, --target_dir TARGET_DIR
+                        target directory where fits files should be copied to
+  -T TARGET_HOST, --target_host TARGET_HOST
+                        target host server where fits files should be copied to
+  -s SAME_ESA_NAME_DIR, --same_esa_name_dir SAME_ESA_NAME_DIR
+                        target directory where fits files should be copied to if there are any naming conflicts
+                        with already published files
+  -d DB_FILE, --db_file DB_FILE
+                        Path to the history publishing database
+  -w WAITING_PERIOD, --waiting_period WAITING_PERIOD
+                        how long to wait after last file modification before publishing
+  -v INCLUDE_VERSIONS, --include_versions INCLUDE_VERSIONS
+                        what versions should be published
+  -l INCLUDE_LEVELS, --include_levels INCLUDE_LEVELS
+                        what levels should be published
+  -p INCLUDE_PRODUCTS, --include_products INCLUDE_PRODUCTS
```

Comment:
would be nice to show the examples of valid values? Is this a copy past of the running the command with `-h` if so maybe add to the actual argparse config?

Reply:
will fix